src/main/anovos/data_report/report_preprocessing.py

```diff
@@ -300,9 +302,11 @@ def charts_to_objects(spark, idf, list_of_cols='all', drop_cols=[], label_col=No
     cutoffs_path2 = source_path + "/drift_statistics/attribute_binning"
 
     idf_encoded.persist(pyspark.StorageLevel.MEMORY_AND_DISK)
-
+    
     if run_type == "local":
         local_path = master_path
+    elif run_type == "databricks":
+        local_path = "/dbfs/FileStore/tables/report_stats"
     else:
```

Comment:
@ranjanravish Same here, don't silently to an `else` if a user misspells "databricks"

Reply:
Added condition that will raise a ValueError to catch typos